### PR TITLE
Change references to Xprint with Vprint

### PR DESCRIPTION
**Pull request by dwijnand**
... particularly in the help description of other flags.

### CODE REVIEW

**Comment by dwijnand** [comment]
Need to fix the issues MiMa found around scala.reflect.runtime.Settings.

File: src/reflect/scala/reflect/runtime/Settings.scala [hunk 1]
```diff
@@ -62,4 +62,7 @@ private[reflect] class Settings extends MutableSettings {
   val Yrecursion        = new IntSetting(0)
   def isScala212        = true
   private[scala] def isScala213 = true
+
+  @deprecated("Use Vprintpos instead", "2.13.1")
+  val Xprintpos: BooleanSetting = Vprintpos
```
Comment (som-snytt) [inline src/reflect/scala/reflect/runtime/Settings.scala:67]: The awful reflect layer is an excellent reason not to make this otherwise-trivial change.

**Comment by som-snytt** [comment]
Some options have more history than others, so, respecting that, I wouldn't assume that everyone's brains will flip a switch, especially for -X things.

**Comment by dwijnand** [comment]
My primary interest is in fixing all the references to -Xprint in the user-facing usage text of `scalac`.  So I can resubmit just that.  But given we've accepted the change from Xprint to Vprint, should we not finish that change in the code?  Having two different names looks like a worse situation than before, if not.

**Comment by som-snytt** [comment]
I surely agree about user-facing typeface. I just wouldn't bother about under the covers too much.

The scalac docs should be updated to say, "An experimental compiler that will remain Scala 2."

**Comment by dwijnand** [comment]
I still think things should be consistent but, in absence of support for that, I've submitted a less-changy #8149.

**Pull request closed without merging**